src/utils/syntax-highlight.js

```diff
@@ -90,7 +90,7 @@ async function importHighlightFileForLanguage(language) {
         languageFile = await import(
           // See bug https://github.com/webpack/webpack/issues/13865
           /* webpackChunkName: "highlight-js-[request]" */
-          `@/../node_modules/highlight.js/lib/languages/${file}`
+          `highlight-js-alias/lib/languages/${file}`
```

Comment:
Should we be using the new alias in all places where we import files from this package or is this the only import that needed to be explicitly changed to use the alias? Just curious.

I'm guessing just this one since it's the only one using the `@`.

Reply:
Nah. the alias is just for here, because there is a webpack bug that we cannot dynamically import those lang files, so it needs an absolute path.